test/test-applications/integrations/Samples.GraphQL/Program.cs

```diff
@@ -19,8 +35,7 @@ public static void Main(string[] args)
             .UseKestrel(serverOptions =>
                 // Explicitly set AllowSynchronousIO to true since the default changes
                 // between AspNetCore 2.0 and 3.0
-                serverOptions.AllowSynchronousIO = true
-            )
+                serverOptions.AllowSynchronousIO = true)
```

Comment:
Q.: just curious, was this pointed by StyleCop?

Reply:
Yes.